articles/200_migration_guide_from_ros1.md

```diff
@@ -102,6 +103,17 @@ Apply the following changes to use `ament_cmake` instead of `catkin`:
 
 * Replace the invocation of `add_message_files`, `add_service_files` and `generate_messages` with [rosidl_generate_interfaces](https://github.com/ros2/rosidl/blob/master/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake).
 
+  * The first argument is the `${PROJECT_NAME}`
+  * Followed by the list of message filenames, relative to the package root.
+    * It is recommended to compose a list of message files and pass the list to the function for clarity.
+  * The `DEPENDENCIES` argument from `generate_messages` is passed at the end.
```

Comment:
`at the end` is not really specific. The dependencies on other packages is specified using the multi-value-keyword argument `DEPENDENCIES`.